tensorboard/webapp/widgets/line_chart_v2/sub_view/line_chart_axis_view.scss

```diff
@@ -215,3 +215,15 @@ $_border-style: 1px solid mat.get-color-from-palette(mat.$gray-palette, 500);
     transform-origin: center;
   }
 }
+
+// Why this weird hack?
+// The prospective fob area is absolutely positioned over the XAxis.
+// This prevents hover events from reaching the XAxis.
+// The .extent-edit-button used to only appear when the XAxis is hovered.
+.line-chart:has(.horizontal-prospective-area:hover) {
```

Comment:
Can you use :host instead of .line-chart?

Reply:
No, this component is the `line-chart-axis`. I could move this block of CSS to the line chart component but, given the extent-edit-button is located here, this seemed a more logical place.